python/examples/cyl-ellipsoid.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import division
+
+import meep as mp
+from meep.geom import Cylinder, epsilon, Ellipsoid, Vector3, Medium
+from meep.simulation import Mirror, Pml, Simulation, no_size
+from meep.source import GaussianSource, Source
+
+
+def main():
+
+    c = Cylinder(radius=3, material=Medium(epsilon_diag=epsilon(3.5)))
```

Comment:
The 'index' is 3.5 and the permittivity (i.e. epsilon) value is the square of the index. This should therefore be `epsilon(3.5*3.5)`.